api/turing/api/ensembling_job_api_it_test.go

```diff
@@ -0,0 +1,330 @@
+// +build integration
+
+package api
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	merlin "github.com/gojek/merlin/client"
+	mlp "github.com/gojek/mlp/client"
+	"github.com/gojek/turing/api/turing/models"
+	"github.com/gojek/turing/api/turing/service"
+	"github.com/gojek/turing/api/turing/service/mocks"
+	"github.com/stretchr/testify/mock"
+	"gotest.tools/assert"
+)
+
+func generateEnsemblingJobFixtureJSON() string {
```

Comment:
Agreed on `$.infra_config.image_ref` will be removing this as this is only needed to build the OCI image and submit the job to Kubernetes. Will be fixing `$.ensembler_config.spec.ensembler.uri` to the local directory, this should be a fixed location because in the Dockerfile I have copied the artifact into the local file directory.

Reply:
done

Reply:
The other comment about the use of `ensembler_config` is not addressed or commented. Can it be `job_config` or something?

Reply:
Done, have changed this to job_config.